### DISCUSSION TITLE
md-switch, Reflect value change from script

### DISCUSSION THREAD

**Discussion by mond210** [material-components/material-web#5157] [Q&A] [answered]
On using <md-switch>, i tried to change 'selected' value from script.

switch changed its value and changed its external appearance as intended. 

However, when actually click the switch, i found that the appearance does not change.

Is there any way to change swicth value in the script?

**Comment by asyncliz** [comment]
Yes, you can set `document.querySelector('md-switch').selected = true`

It seems to be working in [this playground](https://lit.dev/playground/#gist=f5140a533f8ebff7ae259177943dbf1b), can you provide a reproduction of your problem?

**Comment by datvm** [comment, marked as answer]
Is it happen to be similar to the behavior described in #5147 since you said you change the value programmatically?

**Reply by mond210** [reply]
Yes that's what I faced. Thanx for your detailed issue! I wish it'd be fixed soon.

**Reply by datvm** [reply]
@mond210 I gave two workarounds in the meantime in that issue. You can use them while they fix/remake it.
